src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container header-container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="'Moyo Company Logo"
```

Comment:
The `alt` attribute for the logo image contains an unnecessary single quote at the beginning and could be more descriptive. Consider changing it to something like `Moyo Company Logo`.